### ISSUE TITLE
manufacturers referenced via first word

### ISSUE THREAD

**Issue by xloem** [sebaubuntu-python/sebaubuntu_libs#3]
Only first word of manufacturer is preserved:
https://github.com/sebaubuntu-python/sebaubuntu_libs/blob/b544443ff6e95122e15e7190b375986908fed56e/sebaubuntu_libs/libandroid/device_info.py#L118

At https://developers.google.com/zero-touch/resources/manufacturer-names manufacturer values actually retain their entire string with spaces. If that's okay, it might mean adding quotes to shell scripts, not sure.

**Comment by SebaUbuntu** [comment]
Yeah, the only reason I did that formatting is because I use the manufacturer value for device/$manufacturer/$codename device tree path
I should move this to aospdtgen/twrpdtgen
